libdap2/ncd2dispatch.h

```diff
@@ -158,6 +158,9 @@ NCD2_rename_var(int ncid, int varid, const char *name);
 extern int
 NCD2_var_par_access(int, int, int);
 
+extern int
+NCD2_def_var_fill(int, int, int, const void *);
+
```

Comment:
Again, why was this moved? Seems like at least three files are changing for no real reason. Or am I missing something?

Reply:
OK, never mind. I see that you are moving things outside the #ifdef USE_NETCDF4. Now I get it. ;-)